### PR TITLE
[zh-cn]: update the translation of css `color-contrast()` function

### PR DESCRIPTION
**Pull request by T34-active**
### Description
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/color-contrast
### Related issues and pull requests
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/css/color_value/color-contrast/index.md
* Last commit: https://github.com/mdn/content/commit/8a22e494736dbe7cc8ba38127a0d928b9fe8e700

### CODE REVIEW

File: files/zh-cn/web/css/color_value/color-contrast/index.md [hunk 1]
```diff
@@ -1,11 +1,13 @@
 ---
 title: color-contrast()
 slug: Web/CSS/color_value/color-contrast
+l10n:
+  sourceCommit:
```
Comment (T34-active) [inline files/zh-cn/web/css/color_value/color-contrast/index.md:5]:
```suggestion
  sourceCommit: 8a22e494736dbe7cc8ba38127a0d928b9fe8e700
```

File: files/zh-cn/web/css/color_value/color-contrast/index.md [hunk 2]
```diff
@@ -37,3 +39,12 @@ color-contrast(#008080 vs olive, var(--myColor), #d2691e)
...
 {{Compat}}
+
+## 参见
+
+- {{cssxref("color_value", "&lt;color>")}} 数据类型
+- [CSS 颜色](/zh-CN/docs/Web/CSS/CSS_colors)模块
+- [`prefers-contrast`](/zh-CN/docs/Web/CSS/@media/prefers-contrast) 和 [`prefers-color-scheme`](/zh-CN/docs/Web/CSS/@media/prefers-color-scheme) {{cssxref("@media")}} 特性
+- [`contrast()`](/zh-CN/docs/Web/CSS/filter-function/contrast)
+- [WCAG：色彩对比](/zh-CN/docs/Web/Accessibility/Understanding_WCAG/Perceivable/Color_contrast)
+- {{cssxref('--*', 'CSS custom properties')}} 和 {{cssxref("var")}}
```
Comment (T34-active) [inline files/zh-cn/web/css/color_value/color-contrast/index.md:50]:
```suggestion
- {{cssxref('--*', 'CSS 自定义属性')}}和 {{cssxref("var")}}
```

Comment (yin1999) [reply files/zh-cn/web/css/color_value/color-contrast/index.md]: 请改用 markdown 链接，`cssxref` 宏没法通过参数去掉行内代码样式

File: files/zh-cn/web/css/color_value/color-contrast/index.md [hunk 3]
```diff
@@ -37,3 +39,12 @@ color-contrast(#008080 vs olive, var(--myColor), #d2691e)
...
 {{Compat}}
+
+## 参见
+
+- {{cssxref("color_value", "&lt;color>")}} 数据类型
+- [CSS 颜色](/zh-CN/docs/Web/CSS/CSS_colors)模块
+- [`prefers-contrast`](/zh-CN/docs/Web/CSS/@media/prefers-contrast) 和 [`prefers-color-scheme`](/zh-CN/docs/Web/CSS/@media/prefers-color-scheme) {{cssxref("@media")}} 特性
+- [`contrast()`](/zh-CN/docs/Web/CSS/filter-function/contrast)
+- [WCAG：色彩对比](/zh-CN/docs/Web/Accessibility/Understanding_WCAG/Perceivable/Color_contrast)
+- {{cssxref('--*', 'CSS 自定义属性')}}和 {{cssxref("var")}}
```
Comment (T34-active) [inline files/zh-cn/web/css/color_value/color-contrast/index.md:50]:
```suggestion
- [CSS 自定义属性](zh-CN/docs/Web/CSS/--*)和 {{cssxref("var")}}
```

**Review by yin1999 — APPROVED** [review]

**Pull request merged**